src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -292,6 +294,10 @@ object ConductrPlugin extends AutoPlugin {
         conductrImageVersion.toSeq ++
         conductrImage.withFlag(Flags.image) ++
         nrOfContainers.withFlag(Flags.nrOfContainers) ++
+        nrOfInstances.map {
```

Comment:
I feel like having two separate options (nrOfAgents, nrOfCores or similar) would be easier to use and more discoverable, but YMMV.